docs/primitives/a-sky.md

```diff
@@ -4,6 +4,18 @@ type: primitives
 layout: docs
 parent_section: primitives
 source_code: src/extras/primitives/primitives/a-sky.js
+examples:
+  - title:Adding a Sky
+    src: https://glitch.com/edit/#!/ex-1-sky?path=index.html:1:0
+
+ - title:Videosphere Sky
+    src:https://glitch.com/edit/#!/ex-1a-videosphere
+
+ - title:Change Sky
```

Comment:
I'm working on updating this, but we prescribe JS should be contained within components, not raw script tags: https://aframe.io/docs/0.6.0/introduction/javascript-events-dom-apis.html